src/client/hud.cpp

```diff
@@ -862,6 +862,58 @@ void Hud::drawSelectionMesh()
 	}
 }
 
+void Hud::toggleBlockBounds()
+{
+	m_block_bounds_mode++;
+
+	if (m_block_bounds_mode > 2) {
+		m_block_bounds_mode = 0;
+	}
+}
+
+void Hud::drawBlockBounds()
+{
+	if (!m_block_bounds_mode) {
+		return;	
+	}
+
+	video::SMaterial old_material = driver->getMaterial2D();
+	driver->setMaterial(m_selection_material);
+
+	auto pos = player->getStandingNodePos();
```

Comment:
This is not a long type, and statically reading the code is easier with specified types.
```suggestion
	v3s16 pos = player->getStandingNodePos();
```

Reply:
Can you tell I write TypeScript for my day job? 😂